rmgpy/data/kinetics/family.py

```diff
@@ -2172,6 +2171,8 @@ def assignWeightsToEntries(entryNestedList, weightedEntries, N = 1):
                 training = {}
                 
                 for tokenTemplateLabel, weight in weightedEntries:
+                    if 'from training reaction' in tokenTemplateLabel:
```

Comment:
I don't think these changes will work properly with recent changes in uncertainty parsing.  I think you should probably rebase and take a look at the changes.

Reply:
just rebased. no conflicts. `make test` passed. The quote 'from training reaction' is added to kinetics comments on line 1111 of family.py